src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+:root {
+  --blue: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-right: 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.logo {
+  display: flex;
+  justify-self: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+  padding: 0;
+}
+
+.list {
+  display: flex;
+  justify-content: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.list-item {
+  padding-left: 20px;
+}
+
+.list-item:first-child {
+  padding-left: 0;
+}
+
+.link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: black;
+  font-family: "Roboto", sans-serif;
```

Comment:
always set fornt to the `html` and inherit where needed

Reply:
Thank you! Made the changes